api/client/batch_client.py

```diff
@@ -37,6 +39,15 @@ def get_data(self, url, headers, params=None):
         base_log_record = dict(route=url, params=params)
         retry_count = 0
         self._logger.debug(url)
+
+        # append version info
+        headers['python_version'] = platform.python_version()
```

Comment:
better use `-` instead of `_`, see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers

Reply:
updated